src/ekka_mnesia.erl

```diff
@@ -276,7 +276,7 @@ cluster_nodes(stopped) ->
 
 %% @private
 ensure_ok(ok) -> ok;
-ensure_ok({error, {_Node, {already_exists, _Node}}}) -> ok;
+ensure_ok({error, {_, {already_exists, _}}}) -> ok;
```

Comment:
This is not an equivalent change. Is it intended?

Reply:
I'm not sure. reverted.